packages/block-editor/src/components/inserter/test/block-types-tab.native.js

```diff
@@ -52,15 +50,16 @@ describe( 'BlockTypesTab component', () => {
 			( { id, category } ) =>
 				category !== 'reusable' && id !== 'core-embed/a-paragraph-embed'
 		);
-		const component = shallow(
+		const component = render(
 			<BlockTypesTab
 				rootClientId={ 0 }
 				onSelect={ jest.fn() }
-				listProps={ {} }
+				listProps={ { contentContainerStyle: {} } }
 			/>
 		);
-		expect( component.find( BlockTypesList ).prop( 'items' ) ).toEqual(
-			blockItems
-		);
+
+		blockItems.forEach( ( item ) => {
+			component.getByText( item.title );
```

Comment:
I think we should add a `expect` statement here, we could do something like:
```suggestion
			expect( component.getByText( item.title ) ).toBeTruthy();
```